src/thrift-client/thrift_state_generator.cpp

```diff
@@ -593,4 +610,4 @@ soccer::WorldModel ThriftStateGenerator::convertCoachWorldModel(const rcsc::Coac
     res.their_team_score = wm.ourSide() == rcsc::SideID::LEFT ? wm.gameMode().scoreRight() : wm.gameMode().scoreLeft();
     res.is_penalty_kick_mode = wm.gameMode().isPenaltyKickMode();
     return res;
-}
\ No newline at end of file
+}
```

Comment:
Please try not to change the file format, such as adding or removing any spaces or lines.